tools/harness-simulation/README.md

```diff
@@ -11,19 +11,24 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 ## POSIX Environment Setup
 
 1. Build OpenThread to generate standalone OpenThread simulation `ot-cli-ftd`. For example, run the following command in the top directory of OpenThread.
+
```

Comment:
Why adding this new line?

Reply:
The line is removed.

Reply:
It seems that `script/make-pretty markdown` will add the new line.

Reply:
Better use pretty format.